css-three-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  margin: auto;
+}
+
+.padding30 {
+  padding-top: 30px;
+  padding-left: 90px;
+  padding-right: 90px;
+}
+
+.set-background-blue {
+  background-color: rgb(26 150 218);
+}
+
+#page-header {
+  color: white;
+  font-family: Lato, sans-serif;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 0;
+  font-size: 22px;
+}
```

Comment:
I would double-check that all these declarations are actually necessary. This isn't a flexbox, so `justify-content` and `align-items` are likely unnecessary.

Reply:
Rewrote the assignment, and made sure to only use flexbox properties when applicable. I created a .center class for flexbox items and a text-center class for text elements.